federation/cluster/federation-down.sh

```diff
@@ -24,9 +24,29 @@ KUBE_ROOT=$(readlink -m $(dirname "${BASH_SOURCE}")/../../)
 # $KUBEDNS_CONFIGMAP_NAME and $KUBEDNS_CONFIGMAP_NAMESPACE.
 source "${KUBE_ROOT}/federation/cluster/common.sh"
 
+# federation_clusters returns a list of all the clusters in
+# federation, if at all the federation control plane exists
+# and there are any clusters registerd.
+function federation_clusters() {
```

Comment:
Should we really have to hit the API twice? Why not something that only hits the API once?

```bash
if clusters=$("${KUBE_ROOT}/cluster/kubectl.sh" --context="${FEDERATION_KUBE_CONTEXT}" -o jsonpath --template '{.items[*].metadata.name}' get clusters); then
  echo ${clusters}
  return
fi
echo ""
```